metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Installs C compiler / build tools'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '3.2.0'
+version           '3.3.0'
```

Comment:
We shouldn't change the version in this PR. @tas50 and the cookbook engineering team will do that at release time.